### PR TITLE
Docs for 0.26 features

### LINKED ISSUE #5863: Document new features for 0.26
Should cover:
- x-rays/comparisons
- new sync/analysis scheduling options
- binning/histograms
- the revamped new question flow

### PR DESCRIPTION
**Pull request by mazameli**
Includes docs for:
- X-rays and segment:table comparisons
- The revamped New Question flow
- new sync/analysis settings (but not per-field and per-table yet because I'm not sure if it'll make it in time)
- Histograms / binning

Resolves #5863

### CODE REVIEW

File: docs/administration-guide/01-managing-databases.md [hunk 1]
```diff
@@ -57,26 +57,39 @@ To add a database, you'll need its connection information.
...
 
-### Database Analysis
+### Database Sync and Analysis
 
-When Metabase connects to your database, it tries to decipher the field types in your tables based on each field's name. Metabase also takes a sample of each table to look for URL's, json, encoded strings, etc. If a field is classified wrong, you can always manually edit it from the **Metadata** tab in the Admin Panel.
+By default, Metabase performs a lightweight hourly sync of your database, and a nightly deeper analysis of the fields in your tables to power some of Metabase's features, like filter widgets.
 
-### Metadata Syncing
+If you'd like to change these default settings, find on click on your database in the Databases section of the Admin Panel, and turn on the toggle at the bottom of the form that says "This is a large database, so let me choose when Metabase syncs and scans." (This is an option that used to be called "Enable in-depth analysis.")
```
Comment (kdoh) [inline docs/administration-guide/01-managing-databases.md]: I'm assuming `find on click` is meant to be `find and click`?

File: docs/administration-guide/01-managing-databases.md [hunk 2]
```diff
@@ -57,26 +57,39 @@ To add a database, you'll need its connection information.
...
 
-4. Click on the **Sync** button on the right of the screen.
-![databaseconnection](images/DatabaseConnection.png)
+If you'd like to sync your database manually at any time, click on it from the Databases list in the admin panel and click on the Sync button on the right side of the screen:
+
+![Database connection](images/DatabaseConnection.png)
+
+#### Scanning for field values
+
+When Metabase first connects to your database, it tries to guess the types of the fields in your tables based on each field's name. Metabase also takes a sample of each table to look for URLs, JSON, encoded strings, etc. If a field is classified wrong, you can always manually edit it from the **Metadata** tab in the Admin Panel.
```
Comment (kdoh) [inline docs/administration-guide/01-managing-databases.md]: `it tries to guess the types of the fields in your tables based on each fields's name` isn't exactly 100% correct here. It's more a combination of the field's underlying type in the DB, the data, and the field name that influences the types.

File: docs/administration-guide/01-managing-databases.md [hunk 3]
```diff
@@ -57,26 +57,39 @@ To add a database, you'll need its connection information.
...
+
+Metabase also by default performs a more intensive daily sampling of each field's values and caches the distinct values in order to make checkbox and select filters work in dashboards and SQL/native questions. This process can slow down large databases, so if you have a particularly large database, you can turn on the option to choose when Metabase scans, and select one of three scan options in the Scheduling tab:
```
Comment (kdoh) [inline docs/administration-guide/01-managing-databases.md]: `Metabase also by default` feels a bit funny.  `By default, Metabase also performs` might sound a bit more natural?

**Review by kdoh — CHANGES_REQUESTED** [review]
Looks pretty good overall. Left a couple small notes. `/x-ray-data-reference.png` also looks like it has two hover states in the screenshot so we'll want to fix that up as well.

File: docs/administration-guide/01-managing-databases.md (hunk 1 above)
Comment (mazameli) [reply docs/administration-guide/01-managing-databases.md]: Yep, thanks

**Comment by mazameli** [comment]
Good catches, Kyle. Fixed those suggestions and updated the data ref image, so I'll merge in a minute if no objections.

**Review by kdoh — APPROVED** [review]

**Pull request merged**